bioptim/limits/penalty.py

```diff
@@ -467,20 +473,22 @@ def minimize_torque(penalty: PenaltyOption, pn: PenaltyNodes):
             pn: PenaltyNodes
                 The penalty node elements
             """
+            V, first_idx, idx = PenaltyFunctionAbstract._get_states_or_controls_with_specified_var(pn, 'tau')
```

Comment:
Would `_get_variable_from_penalty_node` be a bit more explanative?

Reply:
Please name the variable with a more useful name (`all_tau` for instance instead of `V`). Also, I know we did this in the past, but I am removing them, variables should never use CamelCase, but snake_case

Reply:
Technically, this is not all_tau. V is either all states or all controls. That's why I called it like this. However first_idx, is the first index of the variables we target.

Reply:
The function definitely calls for 'tau' and not 'all'. Unless something very subtle is done by this function, I would expect it to return me the Tau and the Tau only!